src/source/PeerConnection/JitterBuffer.c

```diff
@@ -146,18 +72,36 @@ STATUS jitterBufferPop(PJitterBuffer pJitterBuffer, BOOL bufferClosed)
     CHK(pJitterBuffer->lastPushTimestamp != 0, retStatus);
 
     if (pJitterBuffer->lastPushTimestamp > pJitterBuffer->maxLatency) {
-        earliestTimestamp = pJitterBuffer->lastPushTimestamp - pJitterBuffer->maxLatency;
+        earliestAllowedTimestamp = pJitterBuffer->lastPushTimestamp - pJitterBuffer->maxLatency;
     }
 
-    lastIndex = pJitterBuffer->lastRemovedSequenceNumber;
-    index = pJitterBuffer->lastRemovedSequenceNumber + 1;
+    lastIndex = pJitterBuffer->headSequenceNumber - 1;
+    index = pJitterBuffer->headSequenceNumber;
```

Comment:
What happens if `headSequenceNumber` is `0`.

Reply:
The startIndex will be 0, the lastIndex will be 65535.
It's an abuse of underflow that was also used in the previous iteration.

However I could add an if() clause using MAX_UINT16 if we want to avoid this underflow usage.

Reply:
I'm fine with this.